src/test/java/org/zalando/nakadi/util/HashGeneratorTest.java

```diff
@@ -0,0 +1,65 @@
+package org.zalando.nakadi.util;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+import org.zalando.nakadi.domain.SubscriptionBase;
+import org.zalando.nakadi.view.SubscriptionCursorWithoutToken;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class HashGeneratorTest {
```

Comment:
ok, I added that now https://github.com/zalando/nakadi/pull/635/commits/7a935eda2f2b8a5e9648ab3c8cf3f70ef74f548f
(if this is what you meant :) )